### ISSUE TITLE
margins and padding not working

### ISSUE THREAD

**Issue by atalebagha** [rebassjs/rebass#522]
HI, I'm currently trying to add padding and margin to different components and they don't seem to work.

```javascript
<Box m={8} />
<Box mt={8} />
<Box p={8} />
<Box my={8} />
<Box mx="100px" />
```

None of the above seem to work with any of the components (Flex, Heading, Text, Card etc.)

Thanks!

**Comment by DamickDoni** [comment]
I had the same problem. I fixed it using [rebassjs/grid](https://github.com/rebassjs/grid)

**Comment by jxnblk** [comment]
Thanks, I'll need some more info to reproduce this, if you can provide a codesandbox etc

**Comment by jxnblk** [comment]
Closing this but feel free to reopen if you have a reproducible test case

**Comment by ondrej** [comment]
In case anyone else hits this, seems to be caused by an old version of styled-components. Fixed by upgrading to 3.4.10 (or 4.x).
